app/routes/companyInterest/components/CompanyInterestList.tsx

```diff
@@ -178,6 +233,53 @@ class CompanyInterestList extends Component<Props, State> {
             <Button>Opprett ny bedriftsinteresse</Button>
           </Link>
         </Flex>
+
+        <Flex className={styles.section}>
+          <div className={styles.selector}>
+            <Select
+              name="form-event_selector"
+              value={this.props.selectedEventOption}
+              onChange={this.handleEventChange}
+              options={EVENT_TYPE_OPTIONS}
+              isClearable={false}
+              theme={selectTheme}
+              styles={selectStyles}
+            />
+          </div>
+
+          <div
+            style={{
+              marginTop: '5px',
+            }}
+          >
+            {generatedCSV ? (
+              <a href={generatedCSV.url} download={generatedCSV.filename}>
+                Last ned
+              </a>
+            ) : (
+              <Tooltip
+                style={
+                  this.props.selectedSemesterOption.year && { display: 'none' }
```

Comment:
I feel like adding a `display: none` isn't very "reacty". Just use `!this.props.selectedSemesterOption.year` to determine if the JSX should be rendered.
```js
{!this.props.selectedSemesterOption.year && (
    <Tooltip>
	...
    </Tooltip>
)
```

Reply:
But that would remove the children when the predicate is false. How would you avoid that